OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseTabsDashboardFragment.java

```diff
@@ -99,6 +99,7 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
         courseData = (EnrolledCoursesResponse) requireArguments().getSerializable(Router.EXTRA_COURSE_DATA);
         binding = FragmentCourseTabsDashboardBinding.inflate(inflater, container, false);
+        fragmentItemModels = getFragmentItems();
```

Comment:
use case to move this statment?

Reply:
The case was generated for some hard code values. It wouldn't have been generated for a normal case. 
I will revert it.